chef/cookbooks/hyperv/recipes/config.rb

```diff
@@ -37,13 +37,14 @@
   neutron_service_user = neutron_server[:neutron][:service_user]
   neutron_service_password = neutron_server[:neutron][:service_password]
   neutron_networking_mode = neutron_server[:neutron][:networking_mode]
-  neutron_networking_plugin = neutron_server[:neutron][:networking_plugin]
```

Comment:
This looks wrong. ```networking_mode``` got removed from the neutron barclamp.  And ```networking_plugin``` is still there (can be ```vmware``` or ```ml2``` currently).